x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx

```diff
@@ -313,11 +322,35 @@ export function MlLatencyCorrelations({ onClose }: Props) {
         </EuiFlexItem>
       </EuiFlexGroup>
 
+      {ccsWarning && (
+        <>
+          <EuiSpacer size="m" />
+          <EuiCallOut
+            title={i18n.translate(
+              'xpack.apm.correlations.latencyCorrelations.ccsWarningCalloutTitle',
+              {
+                defaultMessage: 'Cross-cluster search compatilibity.',
```

Comment:
Typo! Needs to be `compatibility`, and I'd remove the `.` as I think this is more in keeping with the writing guidelines.

Reply:
Fixes in 42a7cbd.